assets/js/src/core/components/accordion/accordion.tsx

```diff
@@ -11,24 +11,117 @@
 *  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
 */
 
-import React from 'react'
-import { Collapse, type CollapseProps } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { Collapse, type CollapseProps, Flex } from 'antd'
 import { useStyles } from '@Pimcore/components/accordion/accordion.styles'
+import { type ItemType } from 'rc-collapse/es/interface'
+import i18n from 'i18next'
+import { IconButton } from '@Pimcore/components/icon-button/icon-button'
+
+export interface AccordionItemType extends ItemType {
```

Comment:
Maybe we should also add a theme property? :)

Like success (the greenish one we have for versions), primary (the purple one we have for versions) and for sure some other types. That would make it quite easy to reuse them through the whole app. :)

Reply:
Yes, I wasn't sure if that would be to specific but will add that in now too :)

Reply:
Done :)